docs/conf.py

```diff
@@ -31,6 +31,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
+    'sphinx.ext.doctest',
```

Comment:
For travis we might want to think about building the docs too? Like we do on axelrod we just make sure that `make html` doesn't fail. That helps pickup syntax errors before rtd would tell us about them on a merged branch. In this case it would also run the doctests on travis.